lib/rsolr/client.rb

```diff
@@ -79,7 +80,7 @@ def paginate page, per_page, path, opts = nil
   def update opts = {}
     opts[:headers] ||= {}
     opts[:headers]['Content-Type'] ||= 'text/xml'
-    post 'update', opts
+    post opts.fetch(:update_path, update_path), opts
```

Comment:
What do you think about calling this option just `path`? I see `update` is also called below by a couple other methods, and the connection between `update_path` and, say, `commit` may not be obvious.